src/hotspot/share/prims/scopedMemoryAccess.cpp

```diff
@@ -140,12 +138,11 @@ class CloseScopedMemoryClosure : public HandshakeClosure {
 };
 
 /*
- * This function issues a global handshake operation with all
- * Java threads. This is useful for implementing asymmetric
- * dekker synchronization schemes, where expensive synchronization
- * in performance sensitive common paths, may be shifted to
- * a less common slow path instead.
- * Top frames containg obj will be deoptimized.
+ * This function performs a thread-local handshake against all threads running at the time
```

Comment:
Nit: thread-local??

Reply:
Ha! Mea culpa. I've been too buried in the other kind of thread-local lately. I completely forgot we actually described handshakes that way. :)